app/src/styles/base.css

```
@@ -374,6 +398,33 @@ h1, h2, h3, h4 { font-family: var(--font-heading); margin: 0; }
 .tap { transition: opacity .12s ease; }
 .tap:active { opacity: .6; }
 
+/* The toast: in and out through the bottom edge it lives against, rather
+ * than a fade in place. Enter and exit are the same path in reverse, which
+ * is the whole point — a thing that arrives one way is expected to leave the
+ * way it came. Exit is the shorter of the two, like every other exit here. */
+@keyframes toastIn {
```

Stylelint (17.14.0)

[error] Expected keyframe name "toastIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)